samples/cpp/benchmark_genai/benchmark_genai.cpp

```diff
@@ -36,11 +36,12 @@ int main(int argc, char* argv[]) try {
     size_t num_warmup = result["num_warmup"].as<size_t>();
     size_t num_iter = result["num_iter"].as<size_t>();
   
-    ov::genai::GenerationConfig config;
+    ov::genai::LLMPipeline pipe(models_path, device);
+
+    ov::genai::GenerationConfig config = pipe.get_generation_config();
+    config.do_sample = false;
```

Comment:
some models has beam search by default https://huggingface.co/facebook/blenderbot-3B/blob/main/generation_config.json, so setting do_sample=False is not enough